public/locales/en-US/send.ftl

```diff
@@ -19,66 +20,76 @@ notifyUploadDone = Your upload has finished.
 uploadingFileNotification = Notify me when the upload is complete.
 uploadSuccessConfirmHeader = Ready to Send
 
-uploadSvgAlt.alt = Upload
-
-
-//Note the spec suggests that this string is editable. That feature will not appear at Launch
+uploadSvgAlt
+    .alt = Upload
 uploadSuccessTimingHeader = The link to your file will expire after 1 download or in 24 hours.
 copyUrlFormLabel = Copy and share the link to send your file:
 copyUrlFormLabelWithName = Copy and share the link to send your file: { $filename }
+
 // Note: Title text for button should be the same
 copyUrlFormButton = Copy to clipboard
-  .title = {copyUrlFormButton}
+    .title = Copy to clipboard
 
 copiedUrl = Copied!
 
 deleteFileButton = Delete file
-  .title = {deleteFileButton}
+    .title = Delete file
 sendAnotherFileLink = Send another file
-  .title = {sendAnotherFileLink}
+    .title = Send another file
 
-downloadAltText.alt = Download
+// This alt text and the download filename string are actions.
+downloadAltText
+    .alt = Download
 downloadFileName  = Download { $filename }
+
 downloadFileSize = ({ $size })
 downloadMessage = Your friend is sending you a file with Firefox Send, a service that allows you to share files with a safe, private, and encrypted link that automatically expires to ensure your stuff does not remain online forever.
+
+// This is also an action.
 downloadButtonLabel = Download
-  .title = {downloadButtonLabel}
+    .title = Download
 downloadNotification = Your download has completed.
 downloadFinish = Download Complete
 
+// We might not need to localize this if it's a brand name.
 sendYourFilesLink = Try Firefox Send
-  .title = {sendYourFilesLink}
+    .title = Try Firefox Send
 downloadingPageProgress = Downloading { $filename } ({ $size })
 downloadingPageMessage = Please leave this tab open while we fetch your file and decrypt it.
 
-errorAltText.alt = Upload error
+errorAltText
+    .alt = Upload error
 errorPageHeader = Something went wrong!
 errorPageMessage = There has been an error uploading the file.
 errorPageLink = Send another file
 
 
-linkExpiredAlt.alt = Link expired
+linkExpiredAlt
+    .alt = Link expired
 expiredPageHeader = This link has expired or never existed in the first place!
 notSupportedHeader = Your browser is not supported.
-notSupportedDetail = Unfortunately this browser does not support the web technology that powers Firefox Send. You'll need to try another browser. We recommend Firefox!
+// We might not need to localize the Firefox Send part of this string if it's a brand name.
+notSupportedDetail = Unfortunately this browser does not support the web technology that powers Firefox Send. You’ll need to try another browser. We recommend Firefox!
 downloadFirefoxButtonSub = Free Download
 
 
-sentFilesTitle1 = File
-sentFilesTitle2 = Copy URL
-sentFilesTitle3 = Expires In
-sentFilesTitle4 = Delete
+uploadedFile = File
+copyFileList = Copy URL
+expiryFileList = Expires In
```

Comment:
OK, can you add a comment explaining that? As simple as

// expiryFileList is used as a column header

P.S. I'm in EU timezone, I'll probably catch up next week with this